pkg/apis/application/v1alpha1/types_test.go

```diff
@@ -1435,12 +1425,8 @@ func TestResourceResults_Find(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			got, got1 := tt.r.Find(tt.args.group, tt.args.kind, tt.args.namespace, tt.args.name, tt.args.phase)
-			if got != tt.want {
-				t.Errorf("ResourceResults.Find() got = %v, want %v", got, tt.want)
-			}
-			if !reflect.DeepEqual(got1, tt.want1) {
-				t.Errorf("ResourceResults.Find() got1 = %v, want %v", got1, tt.want1)
-			}
+			assert.Equalf(t, tt.want, got, "ResourceResults.Find() got = %v, want %v", got, tt.want)
+			assert.Truef(t, reflect.DeepEqual(got1, tt.want1), "ResourceResults.Find() got1 = %v, want %v", got1, tt.want1)
```

Comment:
+1, I think we can simplify. Just need a `require.NotNil` and then an `assert.Equal`.